### ISSUE TITLE
Allow a developer to set custom VM Args?

### ISSUE THREAD

**Issue by PiedPieper** [emqtt/emqttd#1492]
#### Environment

- OS: RHEL 7
- Erlang/OTP: 19.2
- EMQ: 2.2

#### Description

Is there any way to add custom vm args? In my other erlang applications, i use a custom rebar3 and app.args file and can declare the setting necessary for distributed erlang:
```
-proto_dist inet_tls
-ssl_dist_opt server_certfile /home/user/certs/server.crt ....
```

In emqttd, you use a .conf file and somehow generate a .args file. In the conf file, there does not seem to be any way to set these options: ## Specify the erlang distributed protocol.

```
##
## Value: Enum
##  - inet_tcp: the default; handles TCP streams with IPv4 addressing.
##  - inet6_tcp: handles TCP with IPv6 addressing.
##
## vm.args: -proto_dist inet_tcp
```

An issue with this is that if you have nodes at multiple data centers and want to cluster/bridge them, all of your communications are insecure and can not only be read but be faked manipulated and cause a plethora of problems because rpcs have no security.

How can a user specify their own VM Args?

**Comment by PiedPieper** [comment]
@emqplus , is there any way that I can accomplish this? How would you go about it?
Thanks

**Comment by emqplus** [comment]
@PiedPieper The issue has been fixed in `develop` branch. More information at PR  #1504 and PR https://github.com/emqtt/emq-docs-en/pull/41.

**Comment by turtleDeng** [comment]
@PiedPieper  Fixed in 2.3.6 release.
